uznum-express/package.json

```diff
@@ -20,11 +21,13 @@
   },
   "dependencies": {
     "ajv": "^8.12.0",
+    "bcrypt": "^5.1.0",
     "config": "^3.3.8",
     "cors": "^2.8.5",
     "couchbase": "^3.2.6",
     "esm": "^3.2.25",
     "express": "^4.18.2",
+    "jsonwebtoken": "^9.0.0",
```

Comment:
`jsonwebtoken` is used to generate JWTs which can be used to ensure the logged in user session. We will store the token on the user's browser and will then use that token to perform certain actions to the backend base on the user's role